lib/apiservers/engine/backends/system.go

```diff
@@ -15,31 +15,148 @@
 package vicbackends
 
 import (
+	"bufio"
 	"fmt"
+	"os"
+	"regexp"
 	"runtime"
+	"strings"
+	"time"
 
 	"golang.org/x/net/context"
 
+	log "github.com/Sirupsen/logrus"
+
+	"github.com/vmware/vic/lib/apiservers/portlayer/client"
+	"github.com/vmware/vic/lib/apiservers/portlayer/client/misc"
+
+	"github.com/docker/docker/pkg/parsers/kernel"
+	"github.com/docker/docker/pkg/platform"
+	"github.com/docker/docker/pkg/system"
 	"github.com/docker/engine-api/types"
 	"github.com/docker/engine-api/types/events"
 	"github.com/docker/engine-api/types/filters"
+	"github.com/docker/go-connections/sockets"
 )
 
 type System struct {
 	ProductName string
 }
 
+type ContainerStatus struct {
+	count      int
+	numRunning int
+	numStopped int
+	numPaused  int
+}
+
+var (
+	etcReleaseFile    = "/etc/os-release"
+	usrlibReleaseFile = "/usr/lib/os-release"
+)
+
 func (s *System) SystemInfo() (*types.Info, error) {
-	Driver := "Portlayer Storage"
-	IndexServerAddress := "https://index.docker.io/v1/"
-	ServerVersion := "0.0.1"
-	Name := "VIC"
+	// Set VIC specific data
+	Driver := s.ProductName + " Backend Engine"
+	IndexServerAddress := "registry-1.docker.io"
+	ServerVersion := "0.3"
```

Comment:
We do.  We need to clean up both errors and strings for both the personality and port layer server.  I don't want to do that with this PR or operation.  I have another GI to clean up strings in the personality server.